app/controllers/api/member_services_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Api::MemberServicesController < ApplicationController
+  # TODO: Authenticate
+
+  def cancel_recurring_donation
+    donations_updater = MemberServicesDonationsUpdater.new(permitted_params.to_h)
+
+    if donations_updater.cancel
+      respond_to do |format|
```

Comment:
I want to render the jbuilder template here

Reply:
If you're only supporting one format, json, then the `respond_to do |format|` can be omitted. 

The `render json: {}` will render inline, converting to json whatever you pass to it. If you want to render a template, it works the same as with other erb/slim templates. `render :action_name` or `render 'template_path'`.

Reply:
I didn't remember how to render jbuilder templates so I looked at app/controllers/api/stateless/members_controller, which also has jbuilder templates and renders them this same way. Have I misunderstood and that controller no longer uses the jbuilder templates?

Reply:
I just looked at the `update` action on the members_controller, and that uses the "implicit" rendering, when successful it never calls render which by default renders the template that's named after the action.

Reply:
My template is also named after the action, so I'm not sure why it doesn't work. Does it only work with the default CRUD actions? I'll try passing the template name.